call-for-submissions.md

```diff
@@ -81,6 +77,6 @@ Here’s how to start the process:
 1. Finally, we publish the piece! Our newsletters currently comes out every other Friday.
 
 
-### Other ways to contribute
+### More ways to contribute
 
-In addition to workers’ perspective, we welcome collaborators who can help with news, history, and songs. We also could use more eyes on proofreading and more hands pushing 'publish' each week! Engaging more collaborators is how we ensure the stewardship of this project for the long-term. Sound interesting to you? [Email us](mailto:twcnewsletter@protonmail.com)!
+In addition to workers’ perspective, we welcome collaborators who can help pull together news, histories, and songs – as well more eyes on proofreading and more hands pushing 'publish' each week! Including collaborators is how we grow our collective and steward this newsletter for the long-term. Sound interesting to you? [Email us](mailto:twcnewsletter@protonmail.com)!
```

Comment:
"histories" sounds weird and i think it's confusing.

```suggestion
In addition to workers’ perspective, we welcome collaborators who can help pull together news, history, and songs &mdash; as well more eyes on proofreading and more hands pushing 'publish' each week! Including collaborators is how we grow our collective and steward this newsletter for the long-term. Sound interesting to you? [Email us](mailto:twcnewsletter@protonmail.com)!
```

Reply:
True. Let's leave it as history.